qemu/tests/drive_mirror.py

```diff
@@ -131,8 +152,6 @@ def wait_for_steady(self):
         """
         params = self.parser_test_args()
         timeout = params.get("wait_timeout")
-        if self.vm.monitor.protocol == "qmp":
```

Comment:
@xutian 
It was my fault, 
cause in drive_mirror_complete.py I add after 'mirror_test.wait_for_steady()' the line 'mirror_test.action_when_steady()' , in which already includes a 'mirror_test.wait_for_steady()', 
so that it loops duplicated, and clearing the event in first loop caused second loop would never get the event.
Would you help review my updates please? Thanks.